Setup/Product.wxs

```diff
@@ -1596,18 +1597,15 @@
         </Feature>
         <Feature Id="Bitbucket" Title="Atlassian Bitbucket Server integration" Level="1">
           <ComponentRef Id="Bitbucket.dll" />
-          <ComponentRef Id="Newtonsoft.Json.dll" />
```

Comment:
Why must these references be removed?

Reply:
The plugins load the DLLs from the app dir. Seems to work.